api/filters/replacement/replacement.go

```diff
@@ -117,6 +118,14 @@ func applyToNode(node *yaml.RNode, value *yaml.RNode, target *types.TargetSelect
 		var t *yaml.RNode
 		var err error
 		if target.Options != nil && target.Options.Create {
+			// create option is not supported in a wildcard matching.
+			// Because, PathMatcher is not supported create option.
+			// So, if create option is set, we fallback to PathGetter.
+			for _, f := range fieldPath {
+				if f == "*" {
+					return errors.New("cannot support create option in a multi-value target")
```

Comment:
You can just add `//nolint:goerr113` to the end of this line. I don't think we need to worry about this dynamic error here since the rest of the code is sprinkled with dynamic errors.

Reply:
Thanks!